logstash/examples/upgrade/test/goss.yaml

```diff
@@ -25,7 +26,8 @@ file:
     filetype: file
     contains:
       - 'http.host: "0.0.0.0"'
-      - 'xpack.monitoring.elasticsearch.hosts: [ "http://elasticsearch:9200" ]'
+      - 'xpack.monitoring.elasticsearch.hosts: [ "https://elasticsearch:9200" ]'
+      - "xpack.monitoring.elasticsearch.ssl.certificate_authority: /usr/share/logstash/config/certs/elastic-certificate.crt"
```

Comment:
This config comes from the default `logstash.yml` file in the Logstash Docker image.
Testing its content isn't anymore relevant.
We should test only that `logstash.yml` file exist with the good permissions.
The contains block can be removed.